app/code/Transoft/Blog/Controller/Adminhtml/Post/Delete.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Controller\Adminhtml\Post;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Transoft\Blog\Model\ResourceModel\Post\CollectionFactory as PostCollectionFactory;
+use Magento\Framework\View\Result\PageFactory;
+
+/**
+ * Delete post controller
+ */
+class Delete extends Action implements HttpGetActionInterface
```

Comment:
Why does **Delete** action is **GET**. Should be **POST**

Reply:
fixed